src/src/Controller/UserController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class UserController extends AbstractFOSRestController
+{
+    /**
+     *
```

Comment:
Still an empty line though :thinking:

Reply:
Huh?

Reply:
Remove the second line of the comment, it's empty